ui/src/main/webapp/src/app/analysis-context/analysis-context-form.component.html

```diff
@@ -89,8 +89,10 @@ <h2 *ngIf="!project" i18n>Loading...</h2>
                     <ng-container i18n>Identifying packages...</ng-container>
                 </p>
             </div>
-            <wu-js-tree-wrapper [treeNodes]="packageTree" [(selectedNodes)]="analysisContext.includePackages">
-            </wu-js-tree-wrapper>
+            <div class="fixed-height">
+                <wu-js-tree-wrapper [treeNodes]="packageTree" [(selectedNodes)]="analysisContext.includePackages">
+                </wu-js-tree-wrapper>
+            </div>
```

Comment:
@mareknovotny @klinki the idea could be to have a parameter `max-height` in the `wu-js-tree-wrapper` to specify if the box must not get bigger than `max-height` value. If there's no value for `max-height` the component can get as big as needed.